torch/nn/quantized/modules/linear.py

```diff
@@ -81,10 +106,10 @@ class Linear(torch.nn.Module):
         >>> print(output.size())
         torch.Size([128, 30])
     """
-    _version = 2
+    _version = 3
```

Comment:
Can you add a comment explaining why the version has changed? I see no change in the class that uses the new version number.

Reply:
In general you're never going to see the current version number: the straightline control flow of the program is the "current" version. You'll only see branches on previous version numbers to adapt some changed functionality